docs/sample/assemblies_library.rst

```diff
@@ -85,22 +85,22 @@ The creation of a :py:class:`SurfactantLayer` object takes a large number argume
     tail_formula = 'C30D64'
     head_thickness = 10.0
     tail_thickness = 16.0
-    head_solvent_surface_coverage = 0.2
-    tail_solvent_surface_coverage = 0.0
+    head_solvation = 0.2
```

Comment:
This is not a good example, as there must be the same number of head and tail groups, so the solvation should be the same